code-list-scripts/cypher/faster-indicators-sic.cypher

```diff
@@ -0,0 +1,51 @@
+CREATE CONSTRAINT ON (n:`_code_faster-indicators-sic`) ASSERT n.code IS UNIQUE;
+CREATE CONSTRAINT ON (n:`_code_list_faster-indicators-sic`) ASSERT n.code IS UNIQUE;
+CREATE (node:`_code_list`:`_code_list_faster-indicators-sic` { label:"sic", edition:"one-off" });
+MERGE (node:`_code`:`_code_faster-indicators-sic` { value:"total" });
+MATCH (parent:`_code_list`:`_code_list_faster-indicators-sic`),                (node:`_code`:`_code_faster-indicators-sic` { value:"total" })                 MERGE (node)-[:usedBy { label:"Total"}]->(parent);
+MERGE (node:`_code`:`_code_faster-indicators-sic` { value:"agriculture-forestry-and-fishing" });
+MATCH (parent:`_code_list`:`_code_list_faster-indicators-sic`),                (node:`_code`:`_code_faster-indicators-sic` { value:"agriculture-forestry-and-fishing" })                 MERGE (node)-[:usedBy { label:"Agriculture, forestry and fishing"}]->(parent);
+MERGE (node:`_code`:`_code_faster-indicators-sic` { value:"production" });
+MATCH (parent:`_code_list`:`_code_list_faster-indicators-sic`),                (node:`_code`:`_code_faster-indicators-sic` { value:"production" })                 MERGE (node)-[:usedBy { label:"Production"}]->(parent);
+MERGE (node:`_code`:`_code_faster-indicators-sic` { value:"construction" });
+MATCH (parent:`_code_list`:`_code_list_faster-indicators-sic`),                (node:`_code`:`_code_faster-indicators-sic` { value:"construction" })                 MERGE (node)-[:usedBy { label:"Construction"}]->(parent);
+MERGE (node:`_code`:`_code_faster-indicators-sic` { value:"services" });
+MATCH (parent:`_code_list`:`_code_list_faster-indicators-sic`),                (node:`_code`:`_code_faster-indicators-sic` { value:"services" })                 MERGE (node)-[:usedBy { label:"Services"}]->(parent);
+MERGE (node:`_code`:`_code_faster-indicators-sic` { value:"a" });
```

Comment:
Dont these SIC categories also have labels that go with them? like A is `Agriculture, Forestry and Fishing` ? so I dont know why some are named and the others are just letters - if they're both used and mean different things then I still think `A - Agriculture, Forestry and Fishing` is more useful than just `A`